src/big/big_int.cr

```diff
@@ -847,6 +855,20 @@ module Math
   def isqrt(value : BigInt)
     BigInt.new { |mpz| LibGMP.sqrt(mpz, value) }
   end
+
+  # Computes the smallest nonnegative power of 2 that is greater than or equal
+  # to *v*.
+  #
+  # The returned value has the same type as the argument.
+  #
+  # ```
+  # Math.pw2ceil(33) # => 64
+  # Math.pw2ceil(64) # => 64
+  # Math.pw2ceil(-5) # => 1
+  # ```
+  def pw2ceil(v : BigInt) : BigInt
```

Comment:
Not a big fan of abbreviations - can this `v` be `value` similar to the 2 methods above?

Reply:
Oh i just saw below you are matching an existing interface :(

Reply:
Yes, that would introduce a breaking change because someone could call the method with a named argument, however unlikely.